components/src/dynamo/trtllm/tests/test_trtllm_video_diffusion.py

```
@@ -390,28 +390,37 @@ class TestVideoData:
 
     def test_empty_defaults(self):
         """Test VideoData with no arguments."""
-        data = VideoData()
+        data = VideoData(response_format="mp4")
+        assert data.response_format == "mp4"
         assert data.url is None
         assert data.b64_json is None
 
+    def test_required_response_format(self):
+        """Test VideoData requires response_format."""
+        with pytest.raises(Exception):  # Pydantic ValidationError
```

Ruff (0.15.10)

[warning] Do not assert blind exception: `Exception`

(B017)